flashcards/src/home.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/main.css" />
+    <title>Flashcards</title>
+  </head>
+  <body>
+    <header class="main-header">
+      <div class="header__content container">
+        <h1 class="text text--xxl text--light text--shadow">Flashcards.com</h1>
+        <div class="header__information">
+          <div class="header__search">
+            <img
+              src="assets/icons/search.svg"
+              alt="search"
+              class="header__search__icon"
+            />
```

Comment:
Any reason to drop the close tag like that @LeVHoangduc ?

Reply:
@hungnguyenhuu-agilityio I have update for this issue